Rakefile

```diff
@@ -135,101 +135,78 @@ end
 REGEN_METADATA = {
     autorest_loc: ENV.fetch('AUTOREST_LOC', '../../../autorest/binaries/net45/AutoRest.exe'),
     azure_sdk: {
-        version: version,
-        tag: 'azure_sdk'
+        version: version
     },
     azure_mgmt_analysis_services: {
-        spec_uri: 'https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-analysisservices/2016-05-16/swagger/analysisservices.json',
+        spec_uri: 'https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current/specification/analysisservices/resource-manager/readme.md',
```

Comment:
same comment as made at another point, I think we should include the tag that we're expecting to generate, being more explicit on the group of files we want, instead of relaying on the default.